doccano_api_client/beta/tests/test_full_integration_tests.py

```diff
@@ -5,7 +5,9 @@
 from ..client import DoccanoClient
 from ..models import Example, Label, Project
 
-DOCCANO_ENDPOINT = "http://0.0.0.0"  # TODO: Make this a pytest parameter
+DOCCANO_ENDPOINT = "http://localhost:8000"  # TODO: Make this a pytest parameter
+DOCCANO_USER = "admin"
+DOCCANO_PASS = "password"
```

Comment:
Can you edit the TODO to make all of these pytest params or ENV variables in the future (actually, that might actually be even better for all three)?

Reply:
@leonardlin bumping on this. When we've knocked it out, I'd love to see your PR on the 1.6/1.7 compatibility bump you mentioned in #72 too!